pandas/io/sql.py

```diff
@@ -419,10 +419,11 @@ def to_sql(frame, name, con, schema=None, if_exists='fail', index=True,
     chunksize : int, default None
         If not None, then rows will be written in batches of this size at a
         time.  If None, all rows will be written at once.
-    dtype : single SQLtype or dict of column name to SQL type, default None
-        Optional specifying the datatype for columns. The SQL type should
-        be a SQLAlchemy type, or a string for sqlite3 fallback connection.
-        If all columns are of the same type, one single value can be used.
+    dtype : dict or scalar, optional
```

Comment:
@jorisvandenbossche FYI this introduces a bit of an incoherent style. If it's agreed that the style in generic is preferred, I can mirror the rest as well.

Reply:
What do you mean exactly with the incoherent style? (the change on the above line looks good)

Reply:
Should've been more explicit... I was referring to `, default None` -> `, optional`.

Reply:
A, that's perfect! (in line with what we wrote down in our docstring guide: http://dev.pandas.io/development/contributing_docstring.html#section-3-parameters)